modules/rostests/winetests/comctl32/pager.c

```diff
@@ -40,8 +40,13 @@ static const WCHAR large_w[] =
     L"You should have received a copy of the GNU Lesser General Public License along with this ...";
 static const WCHAR large_truncated_65_w[65] =
     L"You should have received a copy of the GNU Lesser General Public";
+#ifdef __REACTOS__
+static const WCHAR large_truncated_80_w[80] =
+    L"You should have received a copy of the GNU Lesser General Public License along ";
+#else
 static const WCHAR large_truncated_80_w[80] =
```

Comment:
this is still in latest wine...so

Reply:
huhu

Reply:
wild

Reply:
Therefore @DarkFire01 I wonder whether, with your change (removing the 'w'), what the code intended to test is still working :thinking:

Reply:
It’s not.

Reply:
hmm... ok then, I guess :)

Reply:
Let’s just silence the warning for this file